suite_cam6.xml

```diff
@@ -0,0 +1,361 @@
+      <!-- **************************** -->
+      <scheme>physics_state_check</scheme>      <!-- Only if state_debug_checks is True. This scheme only looks for infs/nans, and kills the model if found. -->
+      <scheme>diag_state_b4_phys_write</scheme> <!-- Diagnostics -->
+      <!-- energy and momentum fixer -->
+      <!-- **************************** -->
+      <scheme>calc_te_and_aam_budgets</scheme>
+      <scheme>check_energy_fix</scheme>                  <!--Only matters if dycore is FV (called "LR" in physics) or SE-->
+      <scheme>check_energy_cam_update_pre_chng</scheme>  <!-- Placeholder for CAM ptend and diagnostic output calculations before "chng" call. -->
+      <scheme>check_energy_chng</scheme>                 <!-- Global integral checker required for certain diagnostic outputs -->
+      <scheme>check_energy_cam_update_post_chng</scheme> <!-- Placeholder for CAM ptend and diagnostic output calculations after "chng" call. -->
+      <scheme>calc_te_and_aam_budgets</scheme>
+      <!-- **************************** -->
+      <scheme>diag_conv_tend_ini</scheme>      <!-- Initalizes convective-scheme diagnostic outputs -->
+      <scheme>calc_dtcore</scheme>             <!-- diagnostic calculation.  Could likely be included in energy fixer scheme. -->
+      <!-- Dry adiabatic adjustment  -->
+      <!-- **************************** -->
+      <!-- Technically only needs temperature and humidity as inputs -->
+      <scheme>dadadj_calc_update</scheme>      <!-- Actual dry adiabatic adjustment scheme, with placeholders for CAM ptend calculations -->
+      <!-- **************************** -->
+      <!-- Zhang-Macfarlane deep convection scheme -->
+      <!-- **************************** -->
+      <scheme>zm_convr</scheme>                 <!-- Actual deep convection routine -->
+      <scheme>zm_convr_cam_update</scheme>      <!-- Placeholder for CAM ptend and diagnostic output calculations -->
+      <scheme>zm_conv_evap</scheme>             <!-- Actual ZM rain evaporation scheme -->
+      <scheme>zm_conv_evap_cam_update</scheme>  <!-- Placeholder for CAM ptend and diagnostic output calculations -->
+      <scheme>momtran</scheme>                  <!-- Actual ZM momentum transport scheme -->
+      <scheme>momtran_cam_update</scheme>       <!-- Placeholder for CAM ptend and diagnostic output calculations -->
+      <scheme>convtran</scheme>                 <!-- Actual ZM convective transport scheme, for constituents in cnst_is_convtran1 -->
+      <scheme>convtran_cam_update</scheme>      <!-- Placeholder for CAM ptend and diagnostic output calculations -->
+      <!-- **************************** -->
+      <scheme>check_energy_chng</scheme>        <!-- Global integral checker required for certain diagnostic outputs -->
+      <scheme>check_tracers_chng</scheme>       <!-- Global tracer mass checker that kills model if error is large enough. -->
```

Comment:
Can you actually update the tracers if they have not been initialized?  (see my previous comment about possibly missing check_tracers_init)

Reply:
Leaving out the `check_tracers_init` scheme was a mistake, although if we treat it as a "timestep_init" routine then its use may be implied, and thus wouldn't have to be included in this file anyways (I think?).